### PR TITLE
Update copy + small fixes

### PR DESCRIPTION
**Pull request by smyzin**
- added loading/completed text for the options
- renamed `STATES` into `States`
- added additional Phase to change card title
- did other minor fixes

### CODE REVIEW

File: src/components/Flow/DecentralizedFlow.tsx [hunk 1]
```diff
@@ -1,44 +1,47 @@
...
+  const { flowState, error } = useSnapshot(AppStore)
   const { jobId } = useSnapshot(JobStore)
   const { subTitle } = SigningStates[flowState]
 
-  const statusDescription = error ? errorList[error] : subTitle
+  const serverCaption = jobId
+    ? ' Feel free to leave and come back—we’ll still be here.'
+    : ''
+  const description = subTitle + serverCaption
+  const statusDescription = error ? errorList[error] : description
```
Comment (T-Damer) [inline src/components/Flow/DecentralizedFlow.tsx:19]: Should we call it `jobStatus` or something like that?

File: src/components/Flow/DecentralizedFlow.tsx [hunk 2]
```diff
@@ -1,44 +1,47 @@
...
   const isError = !!error
 
+  const hasCompleteState = (state: States) => {
+    switch (state) {
+      case States.CHECK_COMMITMENT:
+        return !!AppStore.commitment || !!JobStore.jobId
```
Comment (T-Damer) [inline src/components/Flow/DecentralizedFlow.tsx:25]: Can we include `isCompleted` into `SigningStates` file, so we don't need this `switch`?

File: src/components/Flow/DecentralizedFlow.tsx [hunk 3]
```diff
@@ -1,44 +1,47 @@
...
   const { jobId } = useSnapshot(JobStore)
   const { subTitle } = SigningStates[flowState]
 
-  const statusDescription = error ? errorList[error] : subTitle
+  const serverCaption = jobId
+    ? ' Feel free to leave and come back—we’ll still be here.'
+    : ''
+  const description = subTitle + serverCaption
+  const statusDescription = error ? errorList[error] : description
   const isError = !!error
```
Comment (T-Damer) [inline src/components/Flow/DecentralizedFlow.tsx:20]: Maybe we can mutate `SigningStates` and include an error under specific state, so we don't need `errorList` and will use it like `states[state].error === '' ? undefined : states[state].error`?

File: src/components/Flow/DecentralizedFlow.tsx (hunk 2 above)
Comment (T-Damer) [reply src/components/Flow/DecentralizedFlow.tsx:25]: Like this:
```TS
  [States.CHECK_COMMITMENT]: {
    title: 'Checking for existing commitment...',
    completedText: 'Commitment generated',
    subTitle: 'Hang tight!',
    isCompleted: !!AppStore.commitment || !!JobStore.jobId,
  },
```

**Review by T-Damer — APPROVED** [review]

File: src/components/Flow/DecentralizedFlow.tsx (hunk 1 above)
Comment (smyzin) [reply src/components/Flow/DecentralizedFlow.tsx:19]: Yep, `jobStatus` sounds good for me

File: src/components/Flow/DecentralizedFlow.tsx (hunk 3 above)
Comment (smyzin) [reply src/components/Flow/DecentralizedFlow.tsx:20]: I assume we should separate `state` & `error` cause errors are not rely on states and can be cause in any states or phase

**Review by T-Damer — APPROVED** [review]

**Pull request merged**